plugins/inventory/ovirt.py

```diff
@@ -244,7 +244,7 @@ def parse(self, inventory, loader, path, cache=True):
             username=self.get_option('ovirt_username'),
             password=self.get_option('ovirt_password'),
             ca_file=self.get_option('ovirt_cafile'),
-            insecure=self.get_option('ovirt_insecure'),
+            insecure=not self.get_option('ovirt_cafile'),
```

Comment:
How about to use insecure when specified and if not default to cafile?
```python
insecure=self.get_option('ovirt_insecure') if self.get_option('ovirt_insecure') is not None else not self.get_option('ovirt_cafile')
```

Reply:
I just tested your suggestion, but it appears that ovirt_insecure is being defaulted to false somewhere else.

Reply:
I did a little bit of investigation and the default value is added in the DOCUMENTATION.
I removed the default value and worked as it should.
https://github.com/oVirt/ovirt-ansible-collection/blob/44e9c32b8e47643f95ad142ca6a8dd1a823755f6/plugins/inventory/ovirt.py#L47-L50